modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -1960,6 +1960,33 @@ m|+++false+++
 |===
 
 
+[[config_dbms.cypher.infer_schema_parts]]
+=== `dbms.cypher.infer_schema_parts`
+
+.dbms.cypher.infer_schema_parts
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|Allow label inference during cardinality estimation. During cardinality estimation, if the planner can logically deduce that a node has a label that was not explicitly expressed in the query, the plannerwill use this information during cardinality estimation. +
+This setting controls to what extent the planner should do that:
+
+* `OFF`: No predicates are inferred.
+* `MOST_SELECTIVE_LABEL`: Relationship types are used to infer labels on the relationships' end nodes.
+The label corresponding to the smallest number of nodes is used to estimate rows.
+Not inferring more labels yields better results for nodes with several dependent labels.
```

Comment:
I realise that the [comment](https://github.com/neo-technology/neo4j/pull/25767#discussion_r1626345752) I left on your other PR should have applied to Arne's PR initially.

```suggestion
* `MOST_SELECTIVE_LABEL`: Relationship types are used to infer labels on the relationships' end nodes. The planner will only infer at most one label per node. If more than one label can be inferred for a given node, the planner keeps the most selective one, the one corresponding to the smallest number of nodes in the graph.
```